packages/cms-base/components/SwProductAddToCart.vue

```diff
@@ -36,7 +60,7 @@ const addToCartProxy = async () => {
         class="py-2 px-6 w-full mt-4 bg-gradient-to-r from-cyan-500 to-blue-500 transition ease-in-out hover:bg-gradient-to-l duration-300 cursor-pointer border border-transparent rounded-md flex items-center justify-center text-base font-medium text-white focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
         data-testid="add-to-cart-button"
       >
-        🛍 Add to cart
+        🛍 {{ translations.product.addToCart }}
```

Comment:
please add 🛍 sign to the translation phrase in case someone would not need it ;)

Reply:
But this icon is not related to the translations. Visibility of it should be handled by props